datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/CaveatsDetailsMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.linkedin.datahub.graphql.types.mlmodel.mappers;
+
+import com.linkedin.datahub.graphql.generated.CaveatDetails;
+import com.linkedin.datahub.graphql.types.mappers.ModelMapper;
+
+import lombok.NonNull;
+
+public class CaveatsDetailsMapper implements ModelMapper<com.linkedin.ml.metadata.CaveatDetails, CaveatDetails> {
+
+    public static final CaveatsDetailsMapper INSTANCE = new CaveatsDetailsMapper();
+
+    public static CaveatDetails map(@NonNull final com.linkedin.ml.metadata.CaveatDetails input) {
+        return INSTANCE.apply(input);
+    }
+
+    @Override
+    public CaveatDetails apply(@NonNull final com.linkedin.ml.metadata.CaveatDetails input) {
+        final CaveatDetails result = new CaveatDetails();
+        if (input.getCaveatDescription() != null) {
+            result.setCaveatDescription(input.getCaveatDescription());
```

Comment:
Are these null checks required? These can be skipped if there's no risk of a NullPointerException being thrown

Reply:
(& if the destination field is optional in the GQL schema)